opencti-platform/opencti-front/src/utils/hooks/useQueryLoading.ts

```diff
@@ -21,4 +22,22 @@ const useQueryLoading = <T extends OperationType>(
   return queryRef;
 };
 
+export const useQueryLoadingWithLoadQuery = <T extends OperationType>(
+  query: GraphQLTaggedNode | PreloadableConcreteRequest<T>,
+  variables: VariablesOf<T> = {},
+  opts?: UseQueryLoaderLoadQueryOptions,
+): [useQueryLoaderHookType<T>[0], useQueryLoaderHookType<T>[1]] => {
+  const [queryRef, loadQuery] = useQueryLoader<T>(query);
+  const varRef = useRef(variables);
+  if (!equals(variables, varRef.current)) {
+    varRef.current = variables;
+  }
+  // refetch when variables change
+  useEffect(() => {
+    loadQuery(variables, { ...opts, fetchPolicy: 'store-and-network' });
+  }, [varRef.current]);
+
+  return [queryRef, loadQuery];
+};
```

Comment:
Do we really need to create a  new method here, as the  code is exactly the same as in useQueryLoading except for the return ?